src/Backend/Modules/MediaLibrary/Js/MediaLibraryHelper.js

```diff
@@ -799,6 +799,196 @@ jsBackend.mediaLibraryHelper.group =
     }
 };
 
+jsBackend.mediaLibraryHelper.cropper =
+{
+    cropperQueue: [],
+    isCropping: false,
+
+    passToCropper: function(resizeInfo, resolve, reject) {
+        jsBackend.mediaLibraryHelper.cropper.cropperQueue.push({
+            'resizeInfo': resizeInfo,
+            'resolve': resolve,
+            'reject': reject,
+        });
+
+        // If the cropper is already handling the queue we don't need to start it a second time.
```

Comment:
It's a bit nitpicking, but this looks like the only place where you start your comment with a capital.